lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -99,10 +100,28 @@ def write_playbook
 
       def start_ansible_runner
         command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        setup_verbosity(command) if verbose?
         initialize_command(*command)
         logger.debug("[foreman_ansible] - Running command '#{command.join(' ')}'")
       end
 
+      def setup_verbosity(command)
+        verbosity_level = @verbosity_level.to_i
+        verbosity = '-'
+        verbosity_level.times do
+          verbosity += 'v'
+        end
+        command << verbosity
+      end
+
+      def verbose?
+        verbosity_level = @verbosity_level
+        # rubocop:disable Rails/Present
+        !verbosity_level.nil? && !verbosity_level.empty? &&
+          verbosity_level.to_i.positive?
```

Comment:
If we dont care about explicit 0 being `== nil == ''` what we, as I read, don't, we could simplify and use only one liner method (as `''.to_i == nil.to_i == 0`):
```suggestion
          @verbosity_level.to_i.positive?
```